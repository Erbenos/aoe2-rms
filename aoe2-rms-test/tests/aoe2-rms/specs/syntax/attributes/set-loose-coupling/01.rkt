#lang aoe2-rms
; Give players a group of 7 deer.

<OBJECTS-GENERATION>
(create-object 'DEER
               (number-of-objects 7)
               (number-of-groups 1)
               (group-placement-radius 5)
               (set-loose-grouping)
               (set-gaia-object-only)
               (set-place-for-every-player)
               (min-distance-to-players 14)
               (max-distance-to-players 22))
