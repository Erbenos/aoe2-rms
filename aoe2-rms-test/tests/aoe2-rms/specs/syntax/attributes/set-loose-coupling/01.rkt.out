<OBJECTS_GENERATION>
create_object DEER {
  number_of_objects 7
  number_of_groups 1
  group_placement_radius 5
  set_loose_grouping
  set_gaia_object_only
  set_place_for_every_player
  min_distance_to_players 14
  max_distance_to_players 22
}
